config/database.yml.example

```diff
@@ -84,6 +81,6 @@ test:
 #
 production:
   <<: *default
-  database: proyecto_proy_t03_production
-  username: proyecto_proy_t03
-  password: <%= ENV['PROYECTO_PROY_T03_DATABASE_PASSWORD'] %>
+  database: basedatos_production
+  username: basedatos
+  password: <%= ENV['BASEDATOS_DATABASE_PASSWORD'] %>
```

Comment:
I don't think it is necessary to upload these changes from the sample file for this PR.

Reply:
Hi Yil, right.
Sample file database.yml.example is replaced by database.yml.

Thank you.

Reply:
> Hi Yil, right.
> Sample file database.yml.example is replaced by database.yml.
> 
> Thank you.

The database.yml file is not uploaded to GitHub, it was fine that you were placing the database.yml.example file, my comment was focused on that it was not necessary to make the change in the example file.

Reply:
I agree with @YilKanda here. Changes to this file are not really required, however, I can see this file contained actual database names and usernames. While not a real security risk, I'd rather not have those values inside an `.example` file.